### DISCUSSION TITLE
The "Mute entire site" (forever) option, just like in Chrome when you right-click on tab

### DISCUSSION THREAD

**Discussion by Aosneg** [zen-browser/desktop#2538] [Ideas]
It would be nice to finally have a such feature in Firefox-based browser. 
I just hate that I need to mute some sites manually every time I visit them. And I haven't found extension yet that would entirely solve my problem. Feels like it doesn't exist.

**Comment by jiaweing** [comment]
Just moved to zen and I definitely need this!

**Comment by LogFlames** [comment]
I have the same issue, my school mail keep giving me pings.

Until this is implemented I use this extension, it has both a blacklist (if you want to mute specfic urls) and a whitelist if you wish to mute everything _except_ some urls: https://addons.mozilla.org/en-US/firefox/addon/mutelinks/?src=userprofile

But doing it natively in the browser is preferrable due to the safety risk of 3rd party extensions.
